methods/safe.json

```diff
@@ -5,5 +5,5 @@
   "contactName": "Gnosis Ltd.",
   "contactEmail": "safe@gnosis.io",
   "contactWebsite": "",
-  "specification": "https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-101/CIP-101.md"
-}
\ No newline at end of file
+  "specification": "https://cips.ceramic.network/CIPs/cip-101"
```

Comment:
It is not clear that you are the change controller for this entry. We are sending an email to that address to get confirmation.

Reply:
We should update the contact details to `Safe Ecosystem Foundation` (info@safe.global)

Reply:
Done!